data_safe_haven/serialisers/azure_serialisable_model.py

```diff
@@ -36,16 +36,31 @@ def from_remote_or_create(
         Construct an AzureSerialisableModel from a YAML file in Azure storage, or from
         default arguments if no such file exists.
         """
+        if cls.remote_exists(context):
+            return cls.from_remote(context)
+        else:
+            return cls(**default_args)
+
+    @classmethod
+    def remote_exists(cls: type[T], context: ContextBase) -> bool:
```

Comment:
Is this function used elsewhere?

Reply:
It is also used in the `config upload` command.